modules/psap-using-node-feature-discovery-operator.adoc

```diff
@@ -0,0 +1,176 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/psap-node-feature-discovery-operator.adoc
+
+[id="using-the-node-feature-discovery-operator_{context}"]
+= Using the Node Feature Discovery Operator
+
+The Operator orchestrates all resources needed to run the Node-Feature-Discovery daemon set by watching for a `NodeFeatureDiscovery` CR.
+Based on the `NodeFeatureDiscovery` CR, the Operator will create the operand (NFD) components in the desired namespace, the CR can be edited to choose another `namespace`, `image`, `imagePullPolicy`, and `nfd-worker-conf`, among other options.
+
+As a cluster administrator, you can create a `NodeFeatureDiscovery` instance the {product-title} CLI or the web console.
```

Comment:
"you can create a `NodeFeatureDiscovery` instance the {product-title} CLI or the web console." --> "you can create a `NodeFeatureDiscovery` instance using the {product-title} CLI or the web console."